templates/snap-details.html

```diff
@@ -1,23 +1,23 @@
 {% extends "_layout.html" %}
 
-{% block title %}{{ name }} snap details{% endblock %}
+{% block title %}{{ snap_title }} snap details{% endblock %}
```

Comment:
Not related much to this PR, but maybe it would be good to define how the title should look like (as this is going to be a page linked, indexed by google etc).

As part of snapcraft.io maybe it should have similar title format, something like: `{{ snap_title }} - Snap details - Snaps are universal Linux packages`.
Would be good to confirm this at some point - but not really blocking this PR on that.

Reply:
Yeah I think you should open an issue about that. Or if you think you know what wording would be good just submit a PR with the suggestion.